### PR TITLE
Use webpack global config & @nextcloud/browserslist-config

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -30,15 +30,15 @@ import { generateUrl } from '@nextcloud/router'
...
+const BookmarksApp = (global.Bookmarks = new Vue({
 	el: '#content',
 	store,
 	router,
 	render: h => h(App),
 }))
 
 if ('serviceWorker' in navigator) {
-	navigator.serviceWorker.register(generateUrl('/apps/bookmarks/service-worker.js'))
+	navigator.serviceWorker.register(generateUrl('/apps/bookmarks/js/bookmarks-service-worker.js'))
```
Comment (skjnldsv) [inline src/main.js:41]: Not sure about this, any idea why it was on `/apps/bookmarks` and not `/apps/bookmarks/js` already?

Comment (marcelklehr) [reply src/main.js:41]: This is a hack, because the service worker needs access to all routes under /apps/bookmarks, not just /js.

Comment (skjnldsv) [reply src/main.js:41]: Okay, feel free to adjust then :)

Comment (marcelklehr) [reply src/main.js:41]: Can you revert this?

**Comment by marcelklehr** [comment]
There still seem to be some javascript lint errors.

**Comment by skjnldsv** [comment]
> There still seem to be some javascript lint errors.

done

**Comment by skjnldsv** [comment]
@marcelklehr it takes forever to finish the tests :sweat_smile: 
Some Floccus are failing, can you have a look?

**Comment by marcelklehr** [comment]
yeah, the floccus benchmark sporadically fails that's ok. It's on my todo list.

**Review by marcelklehr — APPROVED** [review]
LGTM :ok_hand:

**Pull request merged**